action.yml

```
@@ -38,6 +38,15 @@ inputs:
       that this is a JSON object containing a key "version".
     type: string
     default: './etc/circt.json'
+  should-create-pr:
+    description: |
+      An optional script to decide whether to open the Pull Request or not.
+      If there is a new version of CIRCT, the action will run the provided
+      script. A return value of 0 means "open the PR". A return value of 1
+      means "do not open the PR". Any other return value will cause the action
+      to exit with an error.
+    type: string
+    default: ''
```

Is there a fixed default value that we can use that does the right thing and is also self-documenting?

```suggestion
    default: 'true'
```